test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/CS/Source/SimpleCodeBlock.cshtml

```diff
@@ -0,0 +1,3 @@
+﻿@{
```

Comment:
rename this to something more specific e.g. `CodeBlockWithTextElement`.  some name that distinguishes this CodeBlock file in terms of what it contains, not the complexity it lacks.